modules/ROOT/pages/endpoint-as2-receive.adoc

```diff
@@ -1,75 +1,81 @@
 = AS2 Receive Endpoint Settings
```

Comment:
Is this really a first-level topic?

Reply:
I think so - these are all field description pages for consistency. They have multiple access points, so I think they are important reference topics that can be accessed or linked to from other topics.